### PR TITLE
Feature/aa 309 add notification about change counterparty verification status

### PR DESCRIPTION
**Pull request by AlexeyShok**
Added notifications for changing activation status of counterparty. Added notifications for changing counterparty vefification state. Fixed unit tests

### CODE REVIEW

File: Api/Models/Mailing/CounterpartyActivityChangedData.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using HappyTravel.Edo.Api.Models.Management.Enums;
+
+namespace HappyTravel.Edo.Api.Models.Mailing
+{
+    public class CounterpartyActivityChangedData : DataWithCompanyInfo
```
Comment (kirillta) [inline Api/Models/Mailing/CounterpartyActivityChangedData.cs:5]: Could this be `CounterpartyStatusChangedData` or `CounterpartyActivityStatusChangedData` to correspond DB field name?

File: Api/AdministratorServices/CounterpartyManagementService.cs [hunk 2]
```diff
@@ -179,6 +204,27 @@ async Task ChangeCounterpartyAccountsActivityStatus()
...
+
+            async Task<Result> SendNotificationToMaster()
+            {
+                var (_, isFailure, master, error) = await GetRootAgencyMasterAgent(counterparty.Id);
+                if (isFailure)
+                    return Result.Failure(error);
+
+                var messageData = new CounterpartyActivityChangedData
+                {
+                    AgentName = $"{master.FirstName} {master.LastName}",
```
Comment (kirillta) [inline Api/AdministratorServices/CounterpartyManagementService.cs:217]: We have so many formatters like this, probably they have to be implemented as a separate method. Just an advise

File: Api/Models/Mailing/CounterpartyActivityChangedData.cs (hunk 1 above)
Comment (AlexeyShok) [reply Api/Models/Mailing/CounterpartyActivityChangedData.cs:5]: This field is named IsActive in the database.

Comment (AlexeyShok) [reply Api/Models/Mailing/CounterpartyActivityChangedData.cs]: Renamed

File: Api/AdministratorServices/CounterpartyManagementService.cs (hunk 2 above)
Comment (AlexeyShok) [reply Api/AdministratorServices/CounterpartyManagementService.cs]: Added get property to model

**Review by kirillta — APPROVED** [review]

**Pull request merged**